src/api/uploads/PlainUpload.js

```diff
@@ -136,7 +136,7 @@ class PlainUpload extends BaseUpload {
         fileDescription: ?string,
         fileId: ?string,
         folderId: string,
-        overwrite: boolean,
+        overwrite: boolean | 'error',
```

Comment:
@coderabbitai where is there a flow error here in the pipeline?

this is the error:
```
string literal `error` [1] is incompatible with boolean [2].

   src/api/uploads/PlainUpload.js:131:9
   131|         overwrite = true,
                ^^^^^^^^^

References:
   src/api/uploads/PlainUpload.js:139:30
   139|         overwrite: boolean | 'error',
                                     ^^^^^^^ [1]
   src/api/uploads/PlainUpload.js:139:20
   139|         overwrite: boolean | 'error',
                           ^^^^^^^ [2]
```

suggest how to fix it

Reply:
well that's lame